modules/mono/glue/GodotSharp/GodotSharp/Core/Vector2.cs

```diff
@@ -603,6 +603,17 @@ public Vector2(Vector2 v)
             y = v.y;
         }
 
+        /// <summary>
+        /// Creates a unit Vector2 rotated to the given angle. This is equivalent to doing
+        /// `Vector2(Mathf.Cos(angle), Mathf.Sin(angle))` or `Vector2.Right.Rotated(angle)`.
+        /// </summary>
+        /// <param name="angle">Angle of the vector, in radians.</param>
+        /// <returns>The resulting vector.</returns>
+        public static Vector2 FromAngle(real_t angle)
+        {
+	        return new Vector2(Mathf.Cos(angle), Mathf.Sin(angle));
```

Comment:
```suggestion
	          return new Vector2(Mathf.Cos(angle), Mathf.Sin(angle));
```

You only have this line intended by 2 spaces instead of 4.

Reply:
Shouldn't it fail static checks? clang-format auto-fixes this 🤔

Reply:
I don't think we have any checks that apply to C#, except for the trivial `file_format.sh`. We should add some checks for the formatting of these files eventually.